app/components/FileLoader.js

```diff
@@ -41,6 +42,24 @@ export default class FileLoader extends Component {
     topNotifOffset: PropTypes.number.isRequired,
   };
 
+  constructor(props) {
+    super(props);
+    this.state = {
+      files: [],
+      selections: [],
+    };
+  }
```

Comment:
Don't think you need to declare a constructor like this, I think you can just define this directly in the class:
```
state = {
 files: [],
 selections: [],
}
```

Though maybe this has limitations that I'm not aware of.

Reply:
as far as I know, it should probably be changed to that yeah